data/Dockerfiles/php-fpm/Dockerfile

```diff
@@ -1,17 +1,62 @@
-FROM php:7.1-fpm-alpine
+FROM alpine:3.6
+
 LABEL maintainer "Andre Peters <andre.peters@servercow.de>"
 
-RUN apk add -U --no-cache libxml2-dev icu-dev icu-libs redis mysql-client bash autoconf g++ make openssl
-RUN pecl install redis && pecl clear-cache
-RUN docker-php-ext-configure intl
-RUN docker-php-ext-install intl pdo pdo_mysql xmlrpc
-RUN docker-php-ext-enable redis
-RUN pear install channel://pear.php.net/Net_IDNA2-0.1.1 Auth_SASL Net_IMAP NET_SMTP Net_IDNA2 Mail_mime
-RUN apk del autoconf g++ make libxml2-dev icu-dev
+# Add script
+COPY docker-entrypoint.sh /
 
-COPY ./docker-entrypoint.sh /
+# Add group + user - 82 is the standard uid/gid for "www-data" in Alpine
+RUN set -x \
+&& addgroup -g 82 -S www-data \
+&& adduser -u 82 -D -S -G www-data www-data \
+\
+# Install Dependencies 
+&& apk add --update \
+&& apk add --no-cache bash curl redis mariadb-client libressl php7-fpm \
```

Comment:
Docker's best practices is to split up packages onto multiple lines and sort it. See [Dockerfile best practices](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#sort-multi-line-arguments)